server/views/pages/opening-times.njk

```diff
@@ -86,16 +86,16 @@
 
 
       <div id="busy" class="{{ {s:12, m:12, l:6, xl:6} | gridClasses }} body-content">
```

Comment:
Shouldn't we rather wrap this up in `body-text`?

Reply:
…or use the `.h2` and `.h3` helper classes?